cmd/ak/cmd/manifest/deploy.go

```diff
@@ -146,26 +147,38 @@ func PrintSuggestions(cmd *cobra.Command, effects manifest.Effects) {
 			continue
 		}
 
-		cid, ok := effect.SubjectID.(sdktypes.ConnectionID)
-		if !ok {
-			continue
-		}
+		if tid, ok := effect.SubjectID.(sdktypes.TriggerID); ok {
+			ctx, cancel := common.LimitedContext()
+			defer cancel()
 
-		ctx, cancel := common.LimitedContext()
-		defer cancel()
+			t, err := common.Client().Triggers().Get(ctx, tid)
+			if err != nil {
+				fmt.Fprintf(cmd.ErrOrStderr(), "error getting trigger %v: %v\n", tid, err)
+				continue
+			}
 
-		c, err := common.Client().Connections().Get(ctx, cid)
-		if err != nil {
-			fmt.Fprintf(cmd.ErrOrStderr(), "error getting connection %v: %v\n", cid, err)
-			continue
+			if slug := t.WebhookSlug(); slug != "" {
+				fmt.Fprintf(cmd.ErrOrStderr(), "[!!!!] Trigger created. Webhook path is \"%s%s\"\n", webhookssvc.WebhooksPathPrefix, slug)
```

Comment:
I prefer this suggestion over adding a missing "." at the end of the line, which is inconsistent with the rest of the output of this command.

```suggestion
				fmt.Fprintf(cmd.ErrOrStderr(), "[!!!!] Trigger created, webhook path is \"%s%s\"\n", webhookssvc.WebhooksPathPrefix, slug)
```

Reply:
Also need to add a trailing "/" after the slug to avoid Go's automatic redirect for ".../foo" to ".../foo/" - because curl doesn't follow redirects automatically.

Reply:
done